### DISCUSSION TITLE
Dashboard Integration - Loveless card or other option?

### DISCUSSION THREAD

**Discussion by Austin1** [music-assistant/support#2859] [Q&A]
I made a great little automation that uses mass.play_media and I can use the standard media card to stop and control this, but I'd like to show the playlist that is playing, which is what play_media feature I'm using on the dashboard itself. Is there anyway to do this today? Any plans for a card? I also tried the spotify playlist card which works, but it doesn't see my sonos devices like music assistant does, so it doesn't work for this usecase.

**Comment by OzGav** [comment]
Duplicate https://github.com/orgs/music-assistant/discussions/954
